app/kumactl/pkg/install/k8s/kuma-cni/templates.go

```diff
@@ -0,0 +1,11 @@
+package kumacni
+
+//go:generate go run github.com/shurcooL/vfsgen/cmd/vfsgendev -source="github.com/Kong/kuma/app/kumactl/pkg/install/k8s/kuma-cni".Templates
```

Comment:
We should have `make generate` for it and test that it is consistent. Check other templates for reference.

We can however do something else. We can keep this `all-in-one-template.yaml` as a separate file and just put this in gigantic if
```
{{if .CNIEnabled}}
all the content of this yaml
{{end}}
```

Reply:
I'm not sure but there might be a scenario when we'd like to have `kumactl install cni-plugin`.  That becomes clearer later when we will implement cni support for all clusters. I'll add `make generate`